File: app/src/androidTest/java/me/kirimin/mitsumine/test/suite/AndroidTestSuite.java
package me.kirimin.mitsumine.test.suite;

import org.junit.runner.RunWith;
import org.junit.runners.Suite;

import me.kirimin.mitsumine.test.EntryInfoApiAccessorTest;
import me.kirimin.mitsumine.test.FeedApiAccessorTest;

@RunWith(Suite.class)
@Suite.SuiteClasses({FeedApiAccessorTest.class, EntryInfoApiAccessorTest.class})
public class AndroidTestSuite {
}

File: app/src/androidTest/java/me/kirimin/mitsumine/test/suite/AndroidTestSuite.kt
package me.kirimin.mitsumine.test.suite

import org.junit.runner.RunWith
import org.junit.runners.Suite

import me.kirimin.mitsumine.test.EntryInfoApiTest
import me.kirimin.mitsumine.test.FeedApiTest

RunWith(Suite::class)
Suite.SuiteClasses(FeedApiTest::class, EntryInfoApiTest::class)
public class AndroidTestSuite